src/tests/ftest/deployment/target_failure.yaml

```diff
@@ -65,7 +65,7 @@ ior: &ior_base
     # takes about 42 sec in the Shared Cluster. Adjust the value based on the system so
     # that it takes longer than the -D value.
     transfer_size: '512K'
-    block_size: '150G'
+    block_size: '100G'
```

Comment:
FYI: reducing block size from 150G to 100G will have no effect, since stonewall is set to 10s. From the log:
https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-8444/21/artifact/Functional%20Hardware%20Medium/deployment/target_failure.py/job.log
`WARNING: Using actual aggregate bytes moved = 47224717312`
And `47224717312 ~= 44G`
Also, `Max Write: 4503.39 MiB/sec`, and `4503.39 MiB * 10 s ~= 44G`.
So whether block size is 150G or 100G, only 44G is actually written. (This relates to my previous explanation of stonewall)

Reply:
I noticed that too. The purpose of this test is to exclude target while IOR is running, not to write the entire `block_size`. It would be better if we can do that after reintegrating the target because that will show that the IOR is fully working. However, that's a little tricky because the amount of `block_size` we can use depends on the pool/storage size. We should also consider the first failed IOR (i.e., does it stop when the target is excluded or does it continue to write even after the exclusion?) I think we should make `block_size` dynamic based on these factors and this will be a future improvement. Anyway, thanks for checking the log.